### ISSUE TITLE
Rendering errors

### ISSUE THREAD

**Issue by kennysong** [HFTrader/DeepLearningBook#1]
A few that I noticed (page numbers are book numbers, not pdf page number):

- Bottom of page 57
- Bottom of page 58
- Bottom of page 60
- Bottom of page 74

**Comment by szrlee** [comment]
A few that I noticed (page numbers are book numbers, not pdf page number):
•Bottom of page 173
•Bottom of page 175
•Bottom of page 181
•Bottom of page 184
etc.

**Comment by HFTrader** [comment]
It seems that these are errors that the author commented - the translation from LATEX to HTML.

**Comment by BAOLONGtrann** [comment]
I have encountered this same problem independently. It seems like you guys used chrome's (or any other blink-based browsers) print-to-pdf function to save the htmls as pdfs. The problems with formatting can be resolved by using gecko-based browser e.g., Firefox.

I have made a new pull request with all the files and the final pdf with bookmark as well. The pdfs rendered by firefox and chrome are a little bit different, that's why I have to commit all pdfs to keep it consistent (instead of just replaced the erred ones).
